packages/reputation-miner/ReputationMinerClient.js

```diff
@@ -4,13 +4,22 @@ const path = require('path');
 
 const ReputationMiner = require("./ReputationMiner");
 
+const minStake = ethers.utils.bigNumberify(10).pow(18).mul(2000); // eslint-disable-line prettier/prettier
```

Comment:
Noting that this is the 3rd place in the `reputation-miner` package where we duplicate these constants.